core/src/main/scala/org/apache/spark/scheduler/BlacklistTracker.scala

```diff
@@ -145,6 +146,72 @@ private[scheduler] class BlacklistTracker (
     nextExpiryTime = math.min(execMinExpiry, nodeMinExpiry)
   }
 
+  private def killBlacklistedExecutor(exec: String): Unit = {
+    if (conf.get(config.BLACKLIST_KILL_ENABLED)) {
+      allocationClient match {
+        case Some(a) =>
+          logInfo(s"Killing blacklisted executor id $exec " +
+            s"since spark.blacklist.killBlacklistedExecutors is set.")
```

Comment:
`${config.BLACKLIST_KILL_ENABLED.key}`
I realize you are just refactoring this, but might as well fix up.